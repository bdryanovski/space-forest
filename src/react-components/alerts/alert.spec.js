import React from 'react'
import renderer from 'react-test-renderer'
import Alert from './alert'

describe('Alert', () => {
  it('should match default snapshot', () => {
    const component = renderer.create(<Alert>Hello</Alert>)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
