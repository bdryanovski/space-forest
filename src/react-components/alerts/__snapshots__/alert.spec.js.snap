// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should match default snapshot 1`] = `
<div
  className="alert"
  role="alert"
>
  <div
    className="alert-close"
    onClick={[Function]}
  >
    Ã—
  </div>
  Hello
</div>
`;
